From 86c41e3ea75d2be6a9ea328bf8eb7644441362f9 Mon Sep 17 00:00:00 2001
From: Newbyte <newbie13xd@gmail.com>
Date: Tue, 2 Feb 2021 13:18:17 +0100
Subject: [PATCH 1/1] explicitly invoke python3 in check-json

python refers to python2 in some distributions which we don't want
---
 tools/check-json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/check-json b/tools/check-json
index de0ee430..6e35395d 100755
--- a/tools/check-json
+++ b/tools/check-json
@@ -3,4 +3,4 @@
 set -e
 
 echo Checking "$1"
-python -m json.tool "$1" > /dev/null || (echo Check failed && exit 255)
+python3 -m json.tool "$1" > /dev/null || (echo Check failed && exit 255)
-- 
2.26.2

